README.md

```diff
@@ -0,0 +1,32 @@
+
+<p align="center">
+<img src="https://cdn-icons-png.flaticon.com/512/4214/4214992.png" width="100px" alt="logo"/>
+</p>
+
+<p align="center">
+  <img src="https://img.shields.io/badge/c%23-%23239120.svg" alt="Csharp CI/CD">
+  <a href="CODE_OF_CONDUCT.md">
+    <img src="https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg" alt="Contributor Covenant">
+  </a>
+  <a href="https://opensource.org/licenses/Apache-2.0">
+    <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License">
+  </a>
+</p>
+
+
+## Foton
+Prototype for an Electron alternative written in C# by Integrativesoft.
+
+## Status of project
+Active
+
+## License
+[Logo](https://www.flaticon.com/br/icone-gratis/foton_4214992) is licensed with Flaticon License - *free for personal and commercial use with attribution.*
```

Comment:
This section is confusing. It seems to be stating that the license of the project is Flaticon, when it's Apache. Here you were trying to give Flaticon the attribution for the logo?